CVLByExample/Reentrancy/README.md

```diff
@@ -12,7 +12,7 @@ The second contract `VulnerableBankEthernaut.sol` is taken from the Ethernaut co
 
 In `VulnerableBankFixed.sol` we show a good way to protect code from reentrancy using reentrancy guards. This ensures that if the contract is reentered, the whole transaction reverts. 
 
-In `VulnerableBankBadFix.sol`, we show an incomplete way to fix the `VulnerableBank.sol` code, and it is conceptually similar to the Curve hack due to the Vyper bad implementation of reentrancy guards https://osec.io/blog/2023-08-01-vyper-timeline. This is because two different guards are used for the `withdraw` and `withdrawAll` functions. This protectsthe contract from reentrancy from one function to itself, but does not protect from cross reentrancy between two external functions. 
+In `VulnerableBankBadFix.sol`, we show an incomplete way to fix the `VulnerableBank.sol` code, and it is conceptually similar to the Curve hack due to the Vyper bad implementation of reentrancy guards https://osec.io/blog/2023-08-01-vyper-timeline. This is because two different guards are used for the `withdraw` and `withdrawAll` functions. This protects the contract from reentrancy from one function to itself, but does not protect from cross reentrancy between two external functions. 
```

Comment:
need to fix also:
"
In the `spec` folder we added a two different specs `Reentrancy.spec` and `ReentrancyEthernaut.spec`" 
we have now 3 files